webpack/assets/javascripts/react_app/components/LoginPage/LoginPageFooter.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { Button } from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../common/I18n';
+
+const LoginPageFooter = ({ caption }) => {
+  const [showLess, setShowLess] = useState(true);
+  if (!caption) return null;
+  let loginCaption = <p>{caption}</p>;
+
+  if (caption.length > 500) {
```

Comment:
Just nit that can be ignored :D

Isn't it better to store such value somewhere in a `constants.js`? Maybe even in a file of general constants, so it can be reused somewhere else?